velox/functions/sparksql/DecimalArithmetic.cpp

```diff
@@ -14,379 +14,71 @@
  * limitations under the License.
  */
 
-#include "velox/common/base/CheckedArithmetic.h"
 #include "velox/expression/DecodedArgs.h"
 #include "velox/expression/VectorFunction.h"
+#include "velox/functions/Macros.h"
```

Comment:
> #include "velox/expression/DecodedArgs.h"
> #include "velox/expression/VectorFunction.h"

Are these includes needed? Would you check if any other includes can be removed?